lib/order.rb

```diff
@@ -0,0 +1,70 @@
+require "csv"
+
+class Order
+  attr_reader :id, :products, :customer
+  attr_accessor :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    raise ArgumentError if ![:pending, :paid, :processing, :shipped, :complete].include?(@fulfillment_status)
+  end
+
+  # collects data into an order
+  def order_info
```

Comment:
I like this method -- when does it get used?

Reply:
If I'm understanding the question I use this to structure what an order type looks like and use the items at the respective indexes in lines 47-55 to read and convert the CSV data into the right types.